keras/src/layers/preprocessing/image_preprocessing/equalization_test.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+import pytest
+from absl.testing import parameterized
+
+from keras.src import layers
+from keras.src import ops
+from keras.src import testing
+
+
+class EqualizationTest(testing.TestCase):
```

Comment:
Please add a test for usage in `tf.data` (similar to other preprocessing / augmentation layers)

Reply:
Okay i will add this test!

Reply:
Done!